api/utils/json.go

```diff
@@ -17,12 +17,41 @@ limitations under the License.
 package utils
 
 import (
+	"bytes"
 	"encoding/json"
+	"unicode"
 
+	"github.com/ghodss/yaml"
 	"github.com/gravitational/configure/jsonschema"
 	"github.com/gravitational/trace"
+	jsoniter "github.com/json-iterator/go"
```

Comment:
Will these be moved out in another PR, seem to be going in the opposite direction we want (adding more dependencies).

Reply:
Yes, these are actually moved back by #5384